guides/2021-05-02-build-flags-backwards-compatibility/go115_en_log.txt

```diff
@@ -0,0 +1,129 @@
+$ go115 version
+go version go1.15.8 linux/amd64
+$ go116 version
+go version go1.16.3 linux/amd64
+$ alias go=go115
+$ mkdir /home/gopher/public
+$ cd /home/gopher/public
+$ go mod init {{{.PUBLIC}}}
+go: creating new go.mod: module {{{.PUBLIC}}}
+$ git init -q
+$ git remote add origin https://{{{.PUBLIC}}}.git
+$ cat <<EOD > /home/gopher/public/public.go
+package public
+
+import (
+    "fmt"
+    "io/ioutil"
+)
+
+func DoSomething() {
+    fmt.Fprintf(ioutil.Discard, "This doesn't print anything")
+}
+EOD
+$ git add public.go go.mod
+$ git commit -q -m 'Initial commit of public module'
+$ git push -q origin main
+remote: . Processing 1 references        
+remote: Processed 1 references in total        
+$ [ "$(git status --porcelain)" == "" ] || (git status && false)
+$ mkdir /home/gopher/gopher
+$ cd /home/gopher/gopher
+$ go mod init gopher
+go: creating new go.mod: module gopher
+$ cat <<EOD > /home/gopher/gopher/gopher.go
+package main
+
+import (
+
+"{{{.PUBLIC}}}"
+)
+
+func main() {
+    public.DoSomething()
+}
+
+EOD
+$ go run .
+go: finding module for package {{{.PUBLIC}}}
+go: downloading {{{.PUBLIC}}} v0.0.0-20210510022515-3b5a503cc6e3
+go: found {{{.PUBLIC}}} in {{{.PUBLIC}}} v0.0.0-20210510022515-3b5a503cc6e3
+$ alias go=go116
+$ cat <<EOD > /home/gopher/public/public.go
+package public
+
+import (
+    "fmt"
+    "io"
+)
+
+func DoSomething() {
+    fmt.Fprintf(io.Discard, "This doesn't print anything")
+}
+EOD
+$ cd /home/gopher/public
+$ git checkout -b bump
```

Comment:
I've tried doing this, the problem is that since I have to do two commit and pushes (break and fix), the proxy cache doesn't resolve the second push fast enough. So the only alternatives that I could came up with was either using the commit (which was going to be a bit difficult) or different branches here. 

Open to ideas on how we can make it work with `@latest` though

Reply:
The canonical way to deal with this is to use `GOPROXY=direct`, as I have done in the commit that I pushed.

Reply:
<3 :thx: